### ISSUE TITLE
Why keep support to python 2?

### ISSUE THREAD

**Issue by cristiansteib** [ArangoDB-Community/pyArango#215]
Python2 has no more support, why this library should keep this compatibility? 

I think the next versions of this library `>= 3` should get rid of python2 support.

Related docs:
- https://www.python.org/doc/sunset-python-2/

**Comment by tariqdaouda** [comment]
I agree with you, we should discuss this with the arangodb people ( @dothebart ). There might be some backcomp issues to keep in mind.

**Comment by dothebart** [comment]
if you want to create a PR for this, me neither thinks we still need this.
